README.md

```
@@ -1,37 +1,74 @@
+- Our changes are in `develop` branch.
+- Latest released package is also `develop` branch, we merge to develop after each of our releases.
+- `master` brach is the original code from Excalidraw as-is. By the time of writing this readme, is unstable and shouldn't be released to production.
+- Some tags are kept in the repo from versions released
+- To the released version number we add the suffix `-alkemio-X` being X the number of our release based on this same Excalidraw's releases, normally just `-alkemio-1`.
+
+
+### Upgrade procedure
+Everytime Excalidraw releases a new package, they publish it in their [GitHub/releases](https://github.com/excalidraw/excalidraw/releases) and they indicate in which commit they have based the release.
+
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)